src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseStatistics.cpp

```diff
@@ -179,6 +182,7 @@ void CompositionalMultiphaseStatistics::computeRegionStatistics( MeshLevel & mes
     regionStatistics.phasePoreVolume.setValues< serialPolicy >( 0.0 );
 
     regionStatistics.phaseMass.setValues< serialPolicy >( 0.0 );
+    regionStatistics.trappedPhaseMass.setValues< serialPolicy >( 0.0 );
```

Comment:
Here and around this call, I think you can replace the `setValues` with `zero`, even in the existing code.